lib/hourglass/redmine_patches/user_preference_patch.rb

```diff
@@ -1,6 +1,11 @@
 module Hourglass
   module RedminePatches
     module UserPreferencePatch
+
+      def self.included(mod)
```

Comment:
As this is a concern, you can use:
```ruby
included do
  safe_attributes 'default_activity'
end
```
maybe needs a self before that `safe attributes`